compatibility_lib/compatibility_lib/test_dependency_highlighter.py

```diff
@@ -144,11 +131,24 @@ def setUp(self):
             "OutdatedDependency<'pip', HIGH_PRIORITY>",
             "OutdatedDependency<'ply', HIGH_PRIORITY>")
 
+        self._store = mock.Mock()
```

Comment:
Maybe it is better to pass in an autospec, otherwise it is possible that you have a typo in your method name, and the test still pass.

Reply:
Done.